source/beamline/CLS/CLSJJSlit.h

```diff
@@ -104,17 +108,35 @@ class CLSJJSlit : public QObject
 	  \param description A human readable description for this JJ slit
 	  \param parent QObject parent class
 	  */
-	CLSJJSlit(const QString &name, const QString &description, const QString &verticalBladesPVBaseName, const QString &horizontalBladesPVBaseName, double tolerance = AMCONTROL_TOLERANCE_DONT_CARE, double moveStartTimeoutSeconds = 2.0, double limit = 11.0, QObject *parent = 0);
+	explicit CLSJJSlit(const QString &name, const QString &description, const QString &upperBladePVName, const QString &lowerBladePVName, const QString &inboardBladePVName, const QString &outboardBladePVName, double tolerance = AMCONTROL_TOLERANCE_DONT_CARE, double moveStartTimeoutSeconds = 2.0, double limit = 11.0, QObject *parent = 0);
```

Comment:
Although it's not required, Creator will almost always add it by default when creating a new class.  I wouldn't worry too much about that @iainworkman.